publisher/internal/analytics/ethereum/analytics.go

```diff
@@ -27,20 +22,26 @@ var eventsToAvoid = []string{
 	"TransformedERC20(address,address,address,uint256,uint256)",
 }
 
+var signaturesToAvoid map[string]struct{}
+
 const (
 	subSubject            = "syntropy.ethereum.log-event"
 	uniswapPositionsOwner = "0xC36442b4a4522E871399CD717aBDD847Ab11FE88"
+	mintSig               = "0x7a53080b" //Mint(address,address,int24,int24,uint128,uint256,uint256)
+	transferSig           = "0xddf252ad" //Transfer(address,address,uint256)
+	burnSig               = "0x0c396cd9" //Burn(address,int24,int24,uint128,uint256,uint256)
 )
 
 type Analytics struct {
 	Options
 	db  repository.Repository
 	ctx context.Context
 
-	eventLogCache     *cache.Cache
-	signatureMint     string
-	signatureTransfer string
-	signaturesToAvoid map[string]struct{}
+	eventLogCache *cache.Cache
+}
+
+func init() {
+	signaturesToAvoid = convertToListOfEventSignatures(eventsToAvoid)
```

Comment:
`convertToListOfEventSignatures` actually returns a map, so maybe there is a better name to reflect that, or even better - omit the return type from the name?